be/src/common/config.h

```diff
@@ -699,6 +699,8 @@ CONF_Int32(metric_late_materialization_ratio, "1000");
 
 // Max batched bytes for each transmit request. (256KB)
 CONF_Int64(max_transmit_batched_bytes, "262144");
+// max chunk size for each tablet write request. (1MB)
+CONF_mInt64(max_tablet_write_chunk_bytes, "1048576");
```

Comment:
need a reasonable default value. @meegoo any suggestions?

Reply:
Too small, unfriendly to wide tables

Reply:
changed to 128MB